thunder/torch/__init__.py

```diff
@@ -5990,6 +5990,102 @@ def mse_loss(
         raise ValueError(f"Reduction argument {reduction} to mse_loss is not supported")
 
 
+if hasattr(torch.nn.functional, "scaled_mm"):
+
+    @torchsymbol(torch.nn.functional.scaled_mm)
+    def scaled_mm(
+        mat_a: TensorLike,
+        mat_b: TensorLike,
+        scale_a,
+        scale_recipe_a,
+        scale_b,
+        scale_recipe_b,
+        swizzle_a=None,
```

Comment:
It would be helpful to add type annotation for `scale_a`, `scale_recipe_a`, `scale_b`, `scale_recipe_b`, `swizzle_a` and `swizzle_b`.